plugins/netskope/plugin.spec.yaml

```diff
@@ -0,0 +1,413 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: netskope
+title: Netskope
+description: Connected with Netskope, this plugin allows to update a file hash list and use it as an allowlist or blocklist for files in your organization, as well as managing URL lists by creating a new URL list or editing existing one. In addition, Netskope plugin has an action that allows to check single user's confidence index
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+supported_versions: ["2022-01-19"]
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/netskope
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.netskope.com/
+tags:
+  - netskope
+  - cloud
+  - security
+hub_tags:
+  use_cases: [cloud_security]
+  keywords: [endpoint]
+  features: []
+enable_cache: false
+connection:
+  tenant:
+    title: Tenant
+    description: Tenant Name
```

Comment:
Try and avoid the first word in the description matching the first word in the title
https://docs.rapid7.com/insightconnect/style-guide/#rules-2

Reply:
Thanks! Changed.